src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -226,18 +227,61 @@
             addItem(itemType, ...options) {
                 let item = getItemDefinition(itemType, this.openmct, this.gridSize, ...options);
                 item.type = itemType;
+                item.id = uuid();
                 this.trackItem(item);
                 this.layoutItems.push(item);
                 this.openmct.objects.mutate(this.internalDomainObject, "configuration.items", this.layoutItems);
                 this.initSelectIndex = this.layoutItems.length - 1;
             },
             trackItem(item) {
                 if (item.type === "telemetry-view") {
-                    this.telemetryViewMap[this.openmct.objects.makeKeyString(item.identifier)] = true;
+                    let keyString = this.openmct.objects.makeKeyString(item.identifier);
+                    let count = this.telemetryViewMap[keyString] || 0;
+                    this.telemetryViewMap[keyString] = ++count;
                 } else if (item.type === "subobject-view") {
                     this.objectViewMap[this.openmct.objects.makeKeyString(item.identifier)] = true;
                 }
             },
+            removeItem(item, index) {
+                this.initSelectIndex = -1;
+                this.layoutItems.splice(index, 1);
+                this.mutate("configuration.items", this.layoutItems);
+                this.untrackItem(item);
+                this.$el.click();
+            },
+            untrackItem(item) {
+                if (item.type === 'telemetry-view') {
+                    let keyString = this.openmct.objects.makeKeyString(item.identifier);
+
+                    if (this.telemetryViewMap[keyString]) {
```

Comment:
this is the level of nesting you can remove by splitting the if statement.  

If you are removing a telemetry-view and it's not in the telemetry view map, then isn't it a programming error that will get silently hidden by this check?  I don't see a valid path where this is undefined?